src/v2/view-builder/views/email/ChallengeAuthenticatorEmailView.js

```diff
@@ -1,57 +1,9 @@
-import { View } from 'okta';
-import hbs from 'handlebars-inline-precompile';
 import BaseAuthenticatorEmailView from './BaseAuthenticatorEmailView';
+import { getCheckYourEmailTitle, getEnterCodeLink } from './AuthenticatorEmailViewUtil';
 
 const BaseAuthenticatorEmailForm = BaseAuthenticatorEmailView.prototype.Body;
 
 // Courage doesn't support HTML, hence creating a subtitle here.
```

Comment:
I think you can move this comment to where you moved the code that was previously under it

Reply:
Done